beat/heart.py

```diff
@@ -1109,39 +1092,77 @@ def plot_waveformfits(self, axes, plotoptions,
         fontsize, syn_color, obs_color, misfit_color, tap_color_annot):
         from beat import plotting
         from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+        import matplotlib.ticker as tick
 
-        def plot_spectrum(
-                axes, data, bbox_to_anchor,
-                lower_corner, upper_corner, syn_color, obs_color, misfit_color, tap_color_annot):
+        def plot_inset_hist(
```

Comment:
why do you redefine that here? its defined in plotting already

Reply:
it was defined in seismic_fits, so I thought it would be different from the other one!

Reply:
no it wasnt, its line 1050 that outside of seismic fits ...

Reply:
please simply use that instead of redefining it

Reply:
> no it wasnt, its line 1050 that outside of seismic fits ...

Line 1708!

Reply:
I guess you have uncommited changes in your local editor? That has been removed maaaaany commits before.